docs/feeds.md

```diff
@@ -0,0 +1,105 @@
+OpenWhisk supports an open API, where any user can expose an event producer service as a **feed** in a **package**.   This section describes architectural and implementation options for providing your own feed.
+
+This material is intended for advanced OpenWhisk users who intend to publish their own feeds.  Most OpenWhisk users can safely skip this section.
+
+# Feed Architecture
+
+There are at least 3 architectural patterns for creating a feed: **Hooks**, **Polling** and **Connections**.
+
+## Hooks
+In the *Hooks* pattern, we set up a feed using a [webhook](https://en.wikipedia.org/wiki/Webhook) facility exposed by another service.   In this strategy, we configure a webhook on an external service to POST directly to a URL to fire a trigger.  This is by far the easiest and most attractive option for implementing low-frequency feeds.
+
+<!-- The github feed is implemented using webhooks.  Put a link here when we have the open repo ready -->
+
+## Polling
+In the "Polling" pattern, we arrange for a OpenWhisk *action* to poll an endpoint periodically to fetch new data.
+This is easy to build, but is low performance, and is limited by the polling interval.
+
+## Connections
+In the "Connections"  pattern, we stand up a separate service somewhere that maintains a persistent connection to a feed source.    The connection based implementation might interact with a service endpoint via long polling, or to set up a push notification.
```

Comment:
Consider choosing a different name for this pattern. Maybe "bridge"? If I understand correctly, the essence is that there is a separate runtime that monitors the external system (one way or another). Type of connection is an implementation detail.